### PR TITLE
fix issue HAWQ-1656 by hardcode installation procedure for bison

### PR DESCRIPTION
**Pull request by ginobiliwang**
This PR is to fix issue 
https://issues.apache.org/jira/browse/HAWQ-1656

### CODE REVIEW

**Comment by radarwave** [comment]
LGTM +1

File: configure [hunk 1]
```diff
@@ -7435,14 +7435,14 @@ if test "$BISON"; then
   pgac_bison_version=`$BISON --version 2>/dev/null | sed q`
   { $as_echo "$as_me:${as_lineno-$LINENO}: using $pgac_bison_version" >&5
 $as_echo "$as_me: using $pgac_bison_version" >&6;}
-  if echo "$pgac_bison_version" | $AWK '{ if ($4 < 1.875) exit 0; else exit 1;}'
+  if echo "$pgac_bison_version" | $AWK '{ if ($4 < 1.875 && $4 > 2.9) exit 0; else exit 1;}'
```
Comment (radarwave) [inline configure]: We should modify configure.in instead of configure, then generate new configure with configure.in.

Comment (ginobiliwang) [reply configure]: fixed in the new commit

**Comment by radarwave** [comment]
+1

**Comment by eschizoid** [comment]
any idea when this will be merged down?

**Comment by radarwave** [comment]
@eschizoid This already been merged to master, please check.

**Comment by benchristel** [comment]
Since this has already been merged, can someone with commit access close this PR please? @lavjain?

**Comment by lavjain** [comment]
@ginobiliwang Please close this PR as only the person who opened the request can close it.

**Pull request closed without merging**